plugins/modules/monitoring/icinga2_host.py

```diff
@@ -211,7 +225,7 @@ def diff(self, hostname, data):
 def main():
     # use the predefined argument spec for url
     argument_spec = url_argument_spec()
-    # remove unnecessary argument 'force'
+    # remove unnecessary argument 'force'  @TODO why is that?
     del argument_spec['force']
```

Comment:
I'm for removing this. Documenting an option that doesn't work is not a good idea.

Next to removing this line, I guess we should add in the documentation above a `force:` entry with `description: ['Not used by this module.']`. That adds this to the option's description.

Reply:
This option comes from `url_argument_spec()` and the docs for it come from the fragments provided for those. If I do what you say, will that override the docs in the fragment?

Reply:
Yes, it will overwrite it. On the other hand, it's not really necessary since the description of `force` already says that it's deprecated and will eventually be removed. Because of that removing `del argument_spec['force']` is even more important, because if the collection is used with ansible-core 2.13+, this module will crash with a `KeyError` when trying to do `del argument_spec['force']`.

Reply:
A separate PR for the monsieur, I presume?

Reply:
It's also fine for me if you fix it here (even without a changelog fragment :) ).

Reply:
I love when you talk dirty :grin: